src/content/docs/alerts/get-notified/muting-rules-suppress-notifications.mdx

```diff
@@ -17,78 +17,88 @@ Alerts sends out timely notifications when your system is having problems. Somet
 
 Once you've spotted the common elements in your unwanted notifications, you can define muting rules that specifically target those elements, while letting other notifications through. Even when a notification is muted, <InlinePopover type="alerts"/> still gathers data on those incidents. Muting rules don't interfere with the alerts process and are applied at the point right before a notification is sent.
 
-## Manage muting rules [#manage]
+## How muting rules work [#overview]
 
-A muting rule condition is the set of individual expressions made up of attributes, operators, and values that define which incidents to target for muting.
+Muting rules are applied at the end of the default alert lifecycle in order to suppress or mute notifications. They don't disable existing policies or conditions. For example, you can mute notifications during known system disruptions, such as maintenance windows and deployments. System disruption incidents will still be identified, even though the notifications for those incidents are muted.
 
-You can create, enable, disable, and manage muting rules. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Muting rules**</DNT>. Enable or disable muting rules at any time. Click the <Icon name="fe-more-horizontal"/> icon on the row of each rule to edit and remove rules.
+A muting rule uses a set of conditions that match against attributes in a [incident event](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/incident-event-attributes). The muting rules tell us how to:
 
-Rules can have one of the following statuses:
+1. Identify individual incidents after they're created, but before an issue is opened.
+2. Override their default condition to indicate that they should be "muted."
 
-* <DNT>**Active**</DNT>: Muting is enabled and active.
-* <DNT>**Scheduled**</DNT>: Muting is enabled but not active yet (there's a future schedule).
-* <DNT>**Ended**</DNT>: Muting is enabled, but no longer active (there's no future schedule).
-* <DNT>**Inactive**</DNT>: Muting is disabled.
+Currently, muting an incident means that the normal alerts incident lifecycle will be maintained, except that an issue that only contains muted incidents will not send out any notifications.
```

Comment:
Alternate: Currently, muting an incident means that the normal alerts incident lifecycle are maintained, except that an issue that only contains muted incidents does not send out any notifications.

Reply:
Thanks! I'm going to change it to this:
`Currently, muting an incident means that the normal alerting incident lifecycle is maintained, except that an issue containing only muted incidents will not send any notifications.`